recipes/libcxxwrap-julia/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cmake -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_INSTALL_LIBDIR=lib $SRC_DIR
+make install
```

Comment:
Is there a `make check` thank can be run, since the tests are only for file existence.

Reply:
I think that the tests are on the julia CxxWrap package which is client to this code.

Quick note on the "make check" note that comes up quite often: I don't think that there is systematically a "check" target. Sometimes tests could be in a "test" or "test" target or anything else.

Reply:
Yeah, I don't think it is systematic. But I thought that cmake generated it automatically

Reply:
Hmmm doesn't that indicate a problem with the recipe then?

Reply:
No I think that it is unrelated to this PR.

libcxxwrap-julia has just been separated out from CxxWrap and the CI and testing scenario has changed quite a bit.

Reply:
Ok! Just making sure